### PR TITLE
Added clone and update scripts

### PR DESCRIPTION
**Pull request by michaldudak**

### CODE REVIEW

**Review by rnowosielski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+An optional `-basePath <path>` parameter can be provided to customize cloning target directory.
+By default, SSH is used to clone the repositories. To use HTTPS, pass in the `cloneWithHttps` flag.
+
+## update
+This scripts automates updating of repository contents using `git pull` command. Usage:
+
+`.\Update-Repos.ps1`
+
+And optional `-basePath <path>` parameter can be provided to customize the base directory of local repositories. The script assumes that all subdirectories of the `basePath` are Git repositories.
```
Comment (rnowosielski) [inline README.md]: I think the readme file doesn't mention the magical project attribute

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Repository Tools
+This directory contains tools useful when operating on projects with multiple Git repositories on Azure Repos.
+These scripts require Powershell Core.
+
+## clone
+This scripts automates cloning of all repositories within a project. Usage:
+
+`.\Clone-Repos.ps1 -project -pat <devops_personal_access_token>`
```
Comment (rnowosielski) [inline README.md]: ok I missed the project :) I thought pat is the switch of token :)

Comment (rnowosielski) [inline README.md]: will it work with `-project` without actually providing project?

Comment (rnowosielski) [reply README.md]: And I guess the organisation is a required parameter too..., right ?

Comment (michaldudak) [reply README.md]: Good catch. I updated it.

**Review by rnowosielski — APPROVED** [review]

**Pull request merged**